### PR TITLE
Disable PushingBox if DHCP failed

### LINKED ISSUE #236: Disable PushingBox if no DHCP
Don't bother trying to use Ethernet!

### PR DESCRIPTION
**Pull request by jgfoster**
Fix #236.

### CODE REVIEW

File: src/TankControllerLib.cpp [hunk 1]
```diff
@@ -79,6 +79,7 @@ TankControllerLib::~TankControllerLib() {
  *
  */
 void TankControllerLib::blink() {
+  // serial("blink() at %li", millis());
```
Comment (prestoncarman) [inline src/TankControllerLib.cpp:82]: Can we remove the commented out code?

File: src/Devices/SD_TC.h [hunk 2]
```diff
@@ -1,7 +1,9 @@
 #pragma once
 
 #include <Arduino.h>
-#include <SD.h>
+
+#define SS 4
```
Comment (prestoncarman) [inline src/Devices/SD_TC.h:5]: Is the define needed?

Comment (jgfoster) [reply src/Devices/SD_TC.h:5]: Yes, see comment in #239.

**Review by prestoncarman — APPROVED** [review]
LGTM

**Pull request merged**